src/styles/landing.scss

```diff
@@ -0,0 +1,63 @@
+nav {
+  .brand-logo {
+    padding-left: 15px !important;
```

Comment:
@Stanley-Okwii Using '!important' is not advised by many CSS experts as it may cause confusion. Could you please change that?

Reply:
@NLSanyu , I have done this for the dom elements that didn't have inline styles. Thank you

Reply:
Great work, @Stanley-Okwii